### ISSUE TITLE
Incompatibility with Phosphophyllite library in 1.18.1

### ISSUE THREAD

**Issue by GordonHollow** [sp614x/optifine#6508]
Many textures of blocks (dirt, stone etc.) just invisible if you turn on any shaders.

Steps to Reproduce
Install Optifine and Phosphophyllite and turn on any shaders

OptiFine-1.18.1_HD_U_H5_pre2
Forge 39.0.6
any version of 1.18 Phosphophyllite

https://github.com/BiggerSeries/Phosphophyllite/issues/10 - issue on their github, author approved incompatibility with Optifine and can't fix it from his side

**Comment by Elrond007** [comment]
Yep, same here, exact same steps to reproduce.

**Comment by woden329** [comment]
Yep - exact same issue here, came across it using a modpack and narrowed it down to Phosphophyllite and OptiFine

**Comment by TheRealAjay** [comment]
Can confirm this, I have the same issue as well.
